src/ICSharpCode.SharpZipLib/Zip/ZipEntry.cs

```diff
@@ -742,7 +742,7 @@ public long DosTime
 					uint mon = Math.Max(1, Math.Min(12, ((uint)(value >> 21) & 0xf)));
 					uint year = ((dosTime >> 25) & 0x7f) + 1980;
 					int day = Math.Max(1, Math.Min(DateTime.DaysInMonth((int)year, (int)mon), (int)((value >> 16) & 0x1f)));
-					DateTime = new DateTime((int)year, (int)mon, day, (int)hrs, (int)min, (int)sec, DateTimeKind.Utc);
+					DateTime = new DateTime((int)year, (int)mon, day, (int)hrs, (int)min, (int)sec, DateTimeKind.Unspecified);
```

Comment:
I don't see any mention of timezones in the appnote, so if it's unknown then unspecified seems a reasonable choice.

Reply:
And more importantly, if you create a DateTime instance and specify that it is in UTC, then the OS will translate it to local time when using it to set the file attributes. If you use `Unspecified` it will not do any such translation.